### PR TITLE
Refactor includes.

### PR DESCRIPTION
**Pull request by siarheishut**
Refactor includes according to https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes.

### CODE REVIEW

File: lib/json/tests/load_test.cpp [hunk 1]
```diff
@@ -1,11 +1,12 @@
-#include <gtest/gtest.h>
-
 #include "json.h"
```
Comment (foxfrikses) [inline lib/json/tests/load_test.cpp:1]: json.h is a library here

File: lib/json/tests/write_test.cpp [hunk 2]
```diff
@@ -1,9 +1,10 @@
-#include <gtest/gtest.h>
-
 #include "json.h"
```
Comment (foxfrikses) [inline lib/json/tests/write_test.cpp:1]: json.h is a library

File: src/main.cpp [hunk 3]
```diff
@@ -1,8 +1,8 @@
+#include <iostream>
+
 #include "json.h"
```
Comment (foxfrikses) [inline src/main.cpp:3]: json is a library

File: src/request_parser.h [hunk 4]
```diff
@@ -1,12 +1,12 @@
 #ifndef ROOT_MANAGER_SRC_REQUEST_PARSER_H_
 #define ROOT_MANAGER_SRC_REQUEST_PARSER_H_
 
-#include "json.h"
-#include "request_types.h"
-
 #include <optional>
 #include <vector>
 
+#include "json.h"
```
Comment (foxfrikses) [inline src/request_parser.h:7]: json is a lib

File: src/request_processor.h [hunk 5]
```diff
@@ -1,11 +1,11 @@
 #ifndef ROOT_MANAGER_SRC_REQUEST_PROCESSOR_H_
 #define ROOT_MANAGER_SRC_REQUEST_PROCESSOR_H_
 
+#include <ostream>
+
 #include "bus_manager.h"
-#include "request_types.h"
 #include "json.h"
```
Comment (foxfrikses) [inline src/request_processor.h:7]: a library

File: src/route_manager.h [hunk 6]
```diff
@@ -1,16 +1,16 @@
...
+#include <optional>
+#include <string>
 #include <unordered_map>
 #include <variant>
 #include <vector>
-#include <string>
 
+#include "common.h"
 #include "request_types.h"
 #include "router.h"
```
Comment (foxfrikses) [inline src/route_manager.h:13]: router is a lib

File: tests/bus_manager_test.cpp [hunk 7]
```diff
@@ -1,11 +1,13 @@
-#include <gtest/gtest.h>
-
-#include "test_utils.h"
+#include "src/bus_manager.h"
```
Comment (foxfrikses) [inline tests/bus_manager_test.cpp:1]: it's not a definition of bus_manager.h, so it should be in the last section

File: tests/request_parser_test.cpp [hunk 8]
```diff
@@ -1,13 +1,14 @@
...
 #include "src/request_parser.h"
-#include "test_utils.h"
 
 #include <string>
 #include <variant>
 #include <vector>
 
+#include "gtest/gtest.h"
+
+#include "json.h"
```
Comment (foxfrikses) [inline tests/request_parser_test.cpp:9]: json is a lib

File: tests/request_parser_test.cpp [hunk 9]
```diff
@@ -1,13 +1,14 @@
-#include <gtest/gtest.h>
-
-#include "json.h"
 #include "src/request_parser.h"
```
Comment (foxfrikses) [inline tests/request_parser_test.cpp:1]: to the last section

File: tests/request_processor_test.cpp [hunk 10]
```diff
@@ -1,10 +1,10 @@
-#include <gtest/gtest.h>
-
 #include "src/request_processor.h"
```
Comment (foxfrikses) [inline tests/request_processor_test.cpp:1]: To the last section

File: lib/json/tests/load_test.cpp (hunk 1 above)
Comment (siarheishut) [reply lib/json/tests/load_test.cpp:1]: Documentation defines such relative order of headers in test file 
```In dir/foo.cc or dir/foo_test.cc, whose main purpose is to implement or test the stuff in dir2/foo2.h, order your includes as follows:

dir2/foo2.h.
A blank line
C system headers (more precisely: headers in angle brackets with the .h extension), e.g., <unistd.h>, <stdlib.h>.
A blank line
C++ standard library headers (without file extension), e.g., <algorithm>, <cstddef>.
A blank line
Other libraries' .h files.
A blank line
Your project's .h files.```

In my situation the main purpose of this file is to test `json.h`, and therefore it seems right to me.

File: lib/json/tests/write_test.cpp (hunk 2 above)
Comment (siarheishut) [reply lib/json/tests/write_test.cpp:1]: Documentation defines such relative order of headers in test file:
In dir/foo.cc or dir/foo_test.cc, whose main purpose is to implement or test the stuff in dir2/foo2.h, order your includes as follows:

dir2/foo2.h.
A blank line
C system headers (more precisely: headers in angle brackets with the .h extension), e.g., <unistd.h>, <stdlib.h>.
A blank line
C++ standard library headers (without file extension), e.g., , .
A blank line
Other libraries' .h files.
A blank line
Your project's .h files.

In my situation the main purpose of this file is to test json.h, and therefore it seems right to me.

File: src/main.cpp (hunk 3 above)
Comment (siarheishut) [reply src/main.cpp:3]: Fixed.

File: src/request_processor.h (hunk 5 above)
Comment (siarheishut) [reply src/request_processor.h:7]: Fixed.

File: src/request_parser.h (hunk 4 above)
Comment (siarheishut) [reply src/request_parser.h:7]: Fixed.

File: src/route_manager.h (hunk 6 above)
Comment (siarheishut) [reply src/route_manager.h]: Fixed.

File: tests/bus_manager_test.cpp (hunk 7 above)
Comment (siarheishut) [reply tests/bus_manager_test.cpp:1]: Documentation defines such relative order of headers in test file:
In dir/foo.cc or dir/foo_test.cc, whose main purpose is to implement or test the stuff in dir2/foo2.h, order your includes as follows:

dir2/foo2.h.
A blank line
C system headers (more precisely: headers in angle brackets with the .h extension), e.g., <unistd.h>, <stdlib.h>.
A blank line
C++ standard library headers (without file extension), e.g., , .
A blank line
Other libraries' .h files.
A blank line
Your project's .h files.

In my situation the main purpose of this file is to test `src/bus_manager.h`, and therefore it seems right to me.

File: tests/request_parser_test.cpp (hunk 8 above)
Comment (siarheishut) [reply tests/request_parser_test.cpp:8]: Fixed.

File: tests/request_parser_test.cpp (hunk 9 above)
Comment (siarheishut) [reply tests/request_parser_test.cpp:1]: Documentation defines such relative order of headers in test file:
In dir/foo.cc or dir/foo_test.cc, whose main purpose is to implement or test the stuff in dir2/foo2.h, order your includes as follows:

dir2/foo2.h.
A blank line
C system headers (more precisely: headers in angle brackets with the .h extension), e.g., <unistd.h>, <stdlib.h>.
A blank line
C++ standard library headers (without file extension), e.g., , .
A blank line
Other libraries' .h files.
A blank line
Your project's .h files.

In my situation the main purpose of this file is to test `src/request_parser.h`, and therefore it seems right to me.

File: tests/request_processor_test.cpp (hunk 10 above)
Comment (siarheishut) [reply tests/request_processor_test.cpp:1]: Documentation defines such relative order of headers in test file:
In dir/foo.cc or dir/foo_test.cc, whose main purpose is to implement or test the stuff in dir2/foo2.h, order your includes as follows:

dir2/foo2.h.
A blank line
C system headers (more precisely: headers in angle brackets with the .h extension), e.g., <unistd.h>, <stdlib.h>.
A blank line
C++ standard library headers (without file extension), e.g., , .
A blank line
Other libraries' .h files.
A blank line
Your project's .h files.

In my situation the main purpose of this file is to test `src/request_processor.h`, and therefore it seems right to me.

File: lib/json/tests/load_test.cpp (hunk 1 above)
Comment (foxfrikses) [reply lib/json/tests/load_test.cpp:1]: You're right.

File: lib/json/tests/write_test.cpp (hunk 2 above)
Comment (foxfrikses) [reply lib/json/tests/write_test.cpp:1]: Right

File: tests/bus_manager_test.cpp (hunk 7 above)
Comment (foxfrikses) [reply tests/bus_manager_test.cpp:1]: Right

**Review by foxfrikses — APPROVED** [review]
LGTM

**Pull request merged**